src/consumer.h

```diff
@@ -12,6 +13,7 @@ typedef struct Consumer{
     int  (*consume) (Consumer c, Message msg);
     void (*consumer_free) (Consumer *c);
     void *meta;
+    Hooklist preadd;
```

Comment:
Is the changed struct somehow a problem during migration?

Reply:
No. Nobody uses a struct as an on disk format, and especially not this struct. There also are no third party libraries, but when there are, a public API freeze is going to happen.